packages/gatsby/src/internal-plugins/functions/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "functions",
   "version": "1.0.0",
   "description": "An internal Gatsby plugin which lets one write serverless functions",
-  "main": "index.js",
+  "main": "index.ts",
```

Comment:
Curious why do we need this change? Are you sure it will work fine in the published package (when all .ts files are removed).

Reply:
So the tests were never including this plugin before because it was experiment. Turning it on for everyone now means that it Is included in our tests for the first time. The tests were failing because the plugin loader couldn't find `index.js` (which was `main` before this change)

In the compiled package it doesn't matter because Gatsby doesn't use `main` to load plugins; it looks for `gatsby-node`

jest (for unit tests on the other hand) were looking for `main`

Hope this makes sense, cheers!

Reply:
Makes sense now. I think alternatively we could have renamed `index.ts` to `index.js` (as it is a no-op anyway). But if you are confident that things will work fine this way - it is enough for me.